expyfun/_input_controllers.py

```diff
@@ -84,6 +99,17 @@ def _on_pyglet_keypress(self, symbol, modifiers, emulated=False):
             this_key = this_key.lstrip('_').lstrip('NUM_')
         self._keyboard_buffer.append((this_key, key_time))
 
+    def _on_pyglet_keyrelease(self, symbol, modifiers, emulated=False):
```

Comment:
I think that the `self.win.on_key_release = self._on_pyglet_keyrelease` should make it get the release events, no?